yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/task/RequestedTextExtractionTaskBuilder.java

```diff
@@ -0,0 +1,39 @@
+package com.yoti.api.client.docs.session.create.task;
+
+/**
+ * Builder to assist creation of {@link RequestedTextExtractionTask}.
+ *
+ * @since 2.7
+ */
+public abstract class RequestedTextExtractionTaskBuilder {
+
+    /**
+     * Sets the text extraction task to always fallback to a manual check
+     *
+     * @return the builder
+     */
+    public abstract RequestedTextExtractionTaskBuilder withManualFallbackAlways();
+
+    /**
+     * Sets the text extraction task to fallback to a manual check if something has gone
+     * wrong with the automated task
+     *
+     * @return the builder
+     */
+    public abstract RequestedTextExtractionTaskBuilder withManualFallbackFallback();
```

Comment:
Do you think we could change the wording then? e.g.:
```suggestion
    public abstract RequestedTextExtractionTaskBuilder withFallbackAsManualFallback();
```

If so, don't mind if we reorder the other ones or not

Reply:
Does `withManualFallbackAlways()` mean it will always trigger a manual check? Could this be `withManualCheck()` or `withManualCheckAlways()`

So we'd have something like this:
- `withManualFallback()`
- `withManualCheck()` or `withManualCheckAlways()`
- `withManualFallbackNever()`

If this is directly reflecting the API, then it's fine as is.

Reply:
Or perhaps this?
- `withManualCheck()` / `withManualCheckAlways()`
- `withManualCheckFallback()`
- `withManualCheckNever()`

Reply:
- withManualCheckAlways()
- withManualCheckFallback()
- withManualCheckNever()

Makes sense to me, and it reflects the JSON. Unless I'm understanding it incorrectly!